### ISSUE TITLE
Feature Request: Gateway docker image

### ISSUE THREAD

**Issue by haragog** [mozilla-iot/gateway#661]
Would be nice to have an easily deployable docker image for testing/running it.

**Comment by thgreasi** [comment]
Hmm, I might be able to tackle this next Friday.

**Comment by rajudev** [comment]
I have started initial work on this issue. Will keep updated about the progress

**Comment by mrstegeman** [comment]
https://github.com/mozilla-iot/gateway-docker

This should work fine after #760 is merged.
